lib/commands/create.js

```diff
@@ -128,6 +113,23 @@ async function main (fabricator, debugOut, argv) {
       tokens,
       specification
     })
+
+    // make sure the cluster has a PX00 mainenance window channel, and the
+    // maintenance startTime is correct
+    const clusterClient = new container.v1.ClusterManagerClient({
+      credentials: cluster.applicationCredentials
+    })
+    const { channels } = await clusterInfo.getCluster(cluster.slug)
+    const win = channels.find(c => c.match(/P\d00/))
```

Comment:
maybe future proof and `/^P\d{3}$/`? anchors too so if we have a channel titled `NLFSTOP123` i don't break everything :joy: